dask_ml/model_selection/_search.py

```diff
@@ -65,8 +65,7 @@
 __all__ = ["GridSearchCV", "RandomizedSearchCV"]
 
 
-if SK_VERSION >= packaging.version.parse("0.19.1"):
-    from sklearn.utils.deprecation import DeprecationDict
+if SK_VERSION <= packaging.version.parse("0.21.dev0"):
```

Comment:
FYI @jrbourbeau aaa0b31f4d894136bec475d2a96018bd808e2dea has your commit bringing in DeprecationDict, and I updated the logic here. I think the correct behavior is

1. Include the train-score keys and warn for 0.20.x
2. Do not include the train-score keys for >=0.21.dev0

Reply:
LGTM, I agree with the logic here